docker-compose.yml

```diff
@@ -41,6 +41,11 @@ services:
       ADAPTIVE_AUTO_ANNOTATION: 'false'
       no_proxy: elasticsearch,kibana,logstash,nuclio,opa,${no_proxy}
       NUMPROCS: 1
+      USE_ALLAUTH_SOCIAL_ACCOUNTS: ${USE_ALLAUTH_SOCIAL_ACCOUNTS}
```

Comment:
It's better to use an empty string "" here, otherwise we'll get a warning `env var is not set...` or something like that